core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2400,38 +2407,42 @@ private void freeAndJournal(RpcContext rpcContext, LockedInodePath inodePath, Fr
     long opTimeMs = System.currentTimeMillis();
     List<Inode<?>> freeInodes = new ArrayList<>();
     freeInodes.add(inode);
+    List<LockedInodePath> descendants = mInodeTree.lockDescendants(inodePath,
```

Comment:
Could you add a comment on why this is not using the `lockDescendants` closable resource, so we know why it does not follow the convention.

Reply:
done